app/src/main/java/edu/uc/willi6jd/mycaloriecounter/FoodActivity.kt

```diff
@@ -26,10 +26,11 @@ class FoodActivity : AppCompatActivity() {
         btnCreate.setOnClickListener {
 
             // Get input fields. Store them to food item variables
-            var itemName = findViewById<EditText>(R.id.foodName); //Name of food item: input text field
-            var itemCalories = findViewById<EditText>(R.id.foodCalories); //Calorie amount of food item: integer
-            var itemSugar = findViewById<EditText>(R.id.foodSugar); //Sugar amount of food item: integer
-            var itemProtein = findViewById<EditText>(R.id.foodProtein); //Protein amount of food: integer
+            var itemName = findViewById<EditText>(R.id.foodName).toString(); //Name of food item: input text field
```

Comment:
You can remove this entirely with Kotlin synthetics.  We used to have to invoke findViewById, which is annoying... with Kotlin synthetics, you can just refer to the component in code, using its identifier in the layout.

Synthetics give you an import that looks something like this:

import kotlinx.android.synthetic.main.main_fragment.*